content/systems/filecoin_token/payment_channels/_index.md

```diff
@@ -2,40 +2,112 @@
 title: Payment Channels
 weight: 2
 bookCollapseSection: true
-dashboardAudit: 0
-dashboardState: wip
+dashboardAudit: 1
+dashboardState: Stable
 dashboardInterface: wip
 ---
 
 # Payment Channels
 ---
 
-Payment Channels are used in the Filecoin [Retrieval Market](retrieval_market) to enable efficient off-chain payments and accounting between parties for what is expected to be series of microtransactions, specifically those occurring as part of retrieval market data retrieval.
+Payment Channels are used in the Filecoin [Retrieval Market](retrieval_market) to enable efficient off-chain payments and accounting between parties for what is expected to be a series of microtransactions, as these occur during data retrieval.
 
-Note that the following provides a high-level overview of payment channels and an accompanying interface. The lotus implementation of [vouchers](https://github.com/filecoin-project/lotus/blob/master/chain/types/voucher.go) and [payment channels](https://github.com/filecoin-project/lotus/tree/master/paych) are also good references.
+In particular, given that there is no proving method provided for the act of sending data from a provider (miner) to a client, there is no trust anchor between the two. Therefore, in order to avoid mis-behaviour, Filecoin is making use of payment channels in order to realise a step-wise "data transfer <-> payment" relationship between the provider and the recipient of data. Clients issue requests for data that miners are responding to. The miner is entitled to ask for interim payments, the volume-oriented interval for which is agreed in the Deal phase. In order to facilitate this process, the Filecoin client is creating a payment channel once the provider has agreed on the proposed deal. The client should also lock in the payment channel monetary value equal to the one needed for retrieval of the entire block of data requested. Every time a provider is completing transfer of the pre-specified amount of data, they can request a payment. The client is responding to this payment with a voucher which the provider can redeem (immediately or later).
```

Comment:
So, I am a bit ambivalent about including this paragraph here.

It's important to note that while Payment Channels are primarily used by Retrieval in the software we've written they are also an independent feature, and they are intended to be a platform for people to write applications that involve payment transfers on top of Filecoin. 

I think the purpose of payment channels is better summarized below:

"The goal of the payment channel actor is to enable a series of off-chain microtransactions to be reconciled on-chain at a later time with fewer messages."

Anyway, this is helpful to understanding how payment and retrieval work together, but I would maybe make a seperate section called "Payment channels and retrieval" to emphasize that payment channels are a feature, and retrieval market is one major use case.